problems/33.Is Subsequence.gs

```diff
@@ -0,0 +1,35 @@
+function isSubsequence(s: string, t: string): boolean {
```

Comment:
I will try to help you what you need to do:
1) It will be enough to just go with for loop though t (bigger string) and just compare with smaller string (s) char by char. When you find match advance one more counter for string s:

if (t[i] == s[j]) {
j++
}

This is just some idea, not fully what you need to write. Use your brain!